android/app/src/main/assets/microlocations

```diff
@@ -1,106 +1,122 @@
 [
-    {
```

Comment:
When updating event from server, all responses should be covered because they depend on each other